recipes/pastrami/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "pastrami" %}
+{% set version = "0.9.5" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://github.com/healthdisparities/pastrami/archive/v{{ version }}.tar.gz"
+  sha256: 4457ad6cda7a2384a1b22109caf3edbff191b3e0a8749f779525276424b4c55c
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - python >=3.8
+    - setuptools
+    - pip
+  run:
+    - python >=3.8
+    - plink2
+    - pathos
+    - numpy
+    - pandas
+    - bzip2
+    - numpy
+    - pandas
+    - pathos
```

Comment:
These two are in here twice. Maybe put `python >=3.8` as the first one and then sort the rest alphabetically?

Reply:
My bad, they shouldn't be repeated